### PR TITLE
Add summit recap post

### PR DESCRIPTION
**Pull request by mosabua**
Pending video uploads and more content - ✅ 

Pending video processing on youtube and making the videos public

Also needs date update just before merge

### CODE REVIEW

File: _posts/2023-12-13-trino-summit-recap.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "Trino Summit 2023 recap"
+author: "Manfred Moser, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2023/summit-logo.png
+show_pagenav: false
+---
+
+Two days of non-stop Trino action completed! Last week Trino Summit 2023
```
Comment (colebow) [inline _posts/2023-12-13-trino-summit-recap.md:10]:
```suggestion
Two days of non-stop Trino action are done! Last week, Trino Summit 2023
```

File: _posts/2023-12-13-trino-summit-recap.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+happened as another great community event. Great presentations from Trino
```
Comment (colebow) [inline _posts/2023-12-13-trino-summit-recap.md:11]: "happened" feels a little passive

File: _posts/2023-12-13-trino-summit-recap.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+experts across the globe showed different use cases and experiences with Trino.
+
+<!--more-->
+
+During the event the lively audience of over 600 attendees asked questions from
```
Comment (colebow) [inline _posts/2023-12-13-trino-summit-recap.md:16]:
```suggestion
During the event, our lively audience of over 600 attendees asked questions from
```

File: _posts/2023-12-13-trino-summit-recap.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+the speakers and each other on chat, and had fun with Trino trivia questions.
```
Comment (colebow) [inline _posts/2023-12-13-trino-summit-recap.md:17]:
```suggestion
the speakers and each other on chat, and we had fun with Trino trivia questions.
```

File: _posts/2023-12-13-trino-summit-recap.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+We talked about the [SQL routine competition]({% post_url 2023-11-09-routines
+%}) and announced Kevin Liu from Stripe and Jan Was from Starburst as the
+winners. You can find their submissions in [the examples page for SQL
+routines]({{site.url}}/docs/current/routine/examples.html).
+
+Starburst announced their [Trino Champions
+program](https://www.starburst.io/community/trino-champions/). Kevin and Jan are
+the first recipients of the award and will receive their swag packs soon. Going
+forward new champions will be crowned regularly, and Starburst is [looking for
```
Comment (colebow) [inline _posts/2023-12-13-trino-summit-recap.md:27]:
```suggestion
forward, new champions will be crowned regularly, and Starburst is [looking for
```

File: _posts/2023-12-13-trino-summit-recap.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+nominations](https://www.starburst.io/community/trino-champions/).
+
+## Sessions
+
+If you missed out on the event, the following list of all the sessions provides
+links to the recordings. Over time we will follow up with blog posts about each
```
Comment (colebow) [inline _posts/2023-12-13-trino-summit-recap.md:33]:
```suggestion
links to the recordings. Over time, we will follow up with blog posts about each
```

File: _posts/2023-12-13-trino-summit-recap.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+  presented by Sönke Liebau ([Stackable](https://stackable.tech/))
+  and Pablo Arteaga ([Bloomberg](https://www.bloomberg.com)).
+* [VAST database catalog]()
+  presented by Jason Russler from
+  [VAST](https://vastdata.com/).
+* [Support for Parquet decryption and aggregate gushdown In Trino]()
+  presented by Amogh Margoor and Manish Malhotra from
+  [Apple](https://www.apple.com/).
+
+Shout out for all their work with the speakers and organizing the event go to
```
Comment (colebow) [inline _posts/2023-12-13-trino-summit-recap.md:92]:
```suggestion
Shout outs for all their work with the speakers and organizing the event go to
```

File: _posts/2023-12-13-trino-summit-recap.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+Anna Schibli, Mandy Darnell, and Monica Miller from the Trino Summit event team,
+and everyone else at Starburst who helped make this event a success.
+
+Special thanks for making Trino Software Foundation event a
```
Comment (colebow) [inline _posts/2023-12-13-trino-summit-recap.md:96]:
```suggestion
Special thanks for making this Trino Software Foundation event a
```

**Review by colebow — CHANGES_REQUESTED** [review]

File: _posts/2023-12-13-trino-summit-recap.md (hunk 2 above)
Comment (mosabua) [reply _posts/2023-12-13-trino-summit-recap.md:11]: took place virtually ? jhahaha


went off ?

**Pull request merged**